config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   root 'home#index'
-  get '/user', to: 'user#index'
+  get '/guitars', to: 'guitars#index'
```

Comment:
Should it be the root route?

Reply:
I thought everyone could see the Gallery page (which could also be the welcome page) 
When the user logs into his account he could add or remove guitars (or only an admin user can do that)
The site could have another welcome page, but I think it would be nicer to see the Gallery when you insert the site
Would that make it the root route? @Ron-Lavi

Reply:
though at the moment if I go to `http://127.0.0.1:3000/` it already shows the guitar app,
I think we can remove the `home_controller` and also `app/views/home/index.html.erb`
and keep only the guitars controller for now.
It's good that we display it as the root page since we don't have login/welcome page

Reply:
Opened an issue on this subject, which would be addressed on a different PR 👍🏼